operations/tempo-mixin/runbook.md

```diff
@@ -310,3 +310,9 @@ This can mean multiple things:
 - scheduler is having issues handing out job or failing internallt due to outage/issues on backend object store.
 
 Look at the logs for scheduler and worker and fix the issues.
+
+## TempoVultureHighErrorRate
+
+This alert fires when Tempo vulture detects a high error rate (above the configured threshold) while validating write or read paths. It indicates there are problems with trace processing or storage.
+
+If a trace write failed once, it will continue to fail on read operations as well. In that case, restarting vulture will start over with new traces.
```

Comment:
> If a trace write failed once, it will continue to fail on read operations as well. In that case, restarting vulture will start over with new traces.

Do we need that? We could just create a ticket to handle this gracefully